docs/troubleshoots.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshoots 
+
+## Setting up Gmail to be used as default mail server in .env
+
+Gmail required applications that will use their service to be authorized first beforehand. Failure in doing this will cause SMTP sending mail process fail with the response **"534-5.7.9 Application-specific password required..."**.
+
+To solve this issue, you need to give this application an App password that is generated from Gmail account.
+
+Here are the steps:
+
+1. Login to your Gmail account and click on **Security** tab from the top navbar.
+![signing_in_settings_gmail_original](https://user-images.githubusercontent.com/29667122/80807468-a1c2ac80-8c00-11ea-9053-7e6759f4c7e6.png)
+
+2. In the section *Signing in to Google* select **2-step verification** and follow the prompts to turn it ON. 
+3. Once step 2 completed, go back to Security page and from the same section (*Signing in to Google*) select **App password**.
+![signing_in_settings_gmail_2steps_on](https://user-images.githubusercontent.com/29667122/80807613-07af3400-8c01-11ea-881c-fb2944e080d4.png)
+
+4. Under *Select the app and device you want to generate the app password for.*:
```

Comment:
When I check the Markdown rendering on the GitHub interface, this step 4 is very confusing. It appears like this:
![image](https://user-images.githubusercontent.com/11148726/80810809-7d44e100-8bbc-11ea-9a86-b45cbfd9d10d.png)

Your markdown needs some adjustments, with the indentation for GitHub to render it well. Instead of this:

![image](https://user-images.githubusercontent.com/11148726/80810880-ab2a2580-8bbc-11ea-8f56-548407ecc6e6.png)

You can fix the identation like this:

![image](https://user-images.githubusercontent.com/11148726/80810994-edebfd80-8bbc-11ea-930a-4dff21c734b0.png)

Reply:
Got it. I've made and pushed the changes. Sorry for this. I wonder if there's a way I could view the .md on IDE before commit and push the changes so to prevent this from happening in the future. For html I normally use the Go Live extension. Do you know an extension I can use to view .md on VSCode, @isabelcosta?

Reply:
No problem @mtreacy002 , we are here learning together. Here's some tools to test Markdown: https://stackedit.io/ and https://markdownlivepreview.com/

For me to see your Markdown rendering I went to GitHub interface, here's how I saw: I clicked on the button next to the Viewed checkbox
![image](https://user-images.githubusercontent.com/11148726/80865656-be9fc400-8c82-11ea-8d67-006f64bd09d7.png)


![image](https://user-images.githubusercontent.com/11148726/80865641-a92a9a00-8c82-11ea-9ece-d6a2d203efbb.png)

Reply:
One more option --> go to those three dots **...** beside of Viewed and then go to **View File** then click it you will be able to view it !!

Reply:
Thanks, @isabelcosta and @robotjellyzone. Those are good options (to view .md on web), but I'm thinking more on to view it before I push the commit to Github. This way I can change things beforehand to avoid multiple commits. But, that's ok, Atom will do for me for now.